packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts

```diff
@@ -194,20 +194,33 @@ export default class JSONSchemaBridge extends Bridge {
         .filter(Boolean);
 
       if (combinedPartials.length) {
-        _definition.properties = definition.properties ?? {};
-        _definition.required = definition.required ?? [];
+        const localProperties = definition.properties
+          ? { ...definition.properties }
+          : {};
+        const localRequired = definition.required
+          ? Array.isArray(definition.required)
+            ? definition.required.slice()
+            : definition.required
+          : [];
```

Comment:
From what I know the `definition.required` can be either array or boolean, so I clone an array or assign the previous boolean value here.

Reply:
No, [it should be an array](https://json-schema.org/understanding-json-schema/reference/object.html#id3).